extension/_locales/en-US/messages.json

```
@@ -1,14 +1,14 @@
 {
     "extensionName": {
       "message": "Firefox Translations",
-      "description": "Name of the extension"
+      "description": "Name of the extension. This name should stay the same for all locales."
     },
     "extensionDescription": {
       "message": "Translate websites in your browser without using the cloud.",
       "description": "Description of the extension."
     },
     "loadingTranslationEngine": {
-      "message": "Loading Translation Engine",
+      "message": "Loading Translation Engine...",
```

```suggestion
      "message": "Loading Translation Engine…",
```

Always proper ellipsis character.